### PR TITLE
Improving the readthedocs documentation

### PR DESCRIPTION
**Pull request by michaeldenes**
This PR is to improve the documentation of `plasticparcels`. Currently a lot of information is missing, specifically the required datasets, and some information on the kernels and initialisation maps.

### CODE REVIEW

File: docs/paper/paper.md [hunk 1]
```diff
@@ -21,41 +21,40 @@ link-bibliography: false
...
...on package for simulating the transport and dispersion of plastics in the ocean. The tool is based on `v3.0.2` of the `Parcels` computational Lagrangian ocean analysis framework [@Lange2017,@Delandmeter2019], providing a modular and customisable collection of methods, notebooks, and tutorials for advecting virtual plastic particles with a wide range of physical properties. The tool applies a collection of physical processes to the virtual particles, such as Stokes drift, wind-induced drift, biofouling, and turbulent mixing, via custom particle behaviour programmed in the form of `Kernels`. In addition to the fine-scale physics parameterisations, `PlasticParcels` provides global particle initialisation maps that represent best estimates for plastic pollution emissions along coastlines [@Jambeck2015], from river sources [@Meijer2021], and in the open-ocean from fishing-related activities [@Kroodsma2018], as well as a current best estimate of buoyant plastic concentrations globally [@Kaandorp2023]. We envisage `PlasticParcels` as a tool for easy-to-run plastic dispersal simulations; as well as for rapid prototyping, development, and testing of new fine-scale physics parameterisations.
```
Comment (erikvansebille) [inline docs/paper/paper.md:24]: use lowercase `plasticparcels` here too?

File: docs/paper/paper.md [hunk 2]
```diff
@@ -21,41 +21,40 @@ link-bibliography: false
...
+`PlasticParcels` has been designed as a layer over the `Parcels` Lagrangian framework [@Lange2017,@Delandmeter2019]. The core functionality of `Parcels` are its `FieldSet`, `Kernel` and `ParticleSet` objects. These objects are designed to be as flexible and customisable as possible so that users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles [@Lange2017]. However, due to the flexible nature of the software, there is a steep learning curve for new users, who often find it difficult to setup their simulations in a rapid fashion due to the complexity of modern hydrodynamic model output. We have developed `PlasticParcels` as user-friendly tool specifically designed for easy-to-generate plastic dispersal simulations. While `PlasticParcels` is primarily designed for use in the cloud and in HPC environments (due to the ever increasing size of hydrodynamic datasets generated from ocean general circulation models), it can be easily installed and run on local machines. A schematic of `PlasticParcels` is shown in Fig. \ref{fig:schematic}.
```
Comment (erikvansebille) [inline docs/paper/paper.md:41]: Also here lowercase `plasticparcels` and `parcels`?

File: docs/paper/paper.md [hunk 3]
```diff
@@ -21,41 +21,40 @@ link-bibliography: false
...
+![Particle initialisations maps based on a) coastal mismanaged plastic waste emissions [@Jambeck2015], b) riverine mismanaged plastic waste emissions [@Meijer2021], c) fishing activity related plastic emissions [@Kroodsma2018], and d) a current global surface concentration estimate [@Kaandorp2023].\label{fig:initialisation_maps}](initialisation_maps.png){width=80%}
 
+In addition, due to the flexibility of the package, users may use the functions and modular design of `PlasticParcels` to enhance their existing `Parcels` simulations and workflow. For example, users can use the initialisation maps, associated `ParticleSet` creation methods, and/or the custom physics kernels with their own `Parcels` simulations. Post-processing and analysis of the generated trajectory datasets is purposefully left to the user, however some tutorials are provided in the [`PlasticParcels` github repository](\url{https://github.com/OceanParcels/PlasticParcels/docs/examples}), along with the tutorials in the [`Parcels` documentation](\url{https://docs.oceanparcels.org/en/latest/documentation.html}). Below we provide an example use case of `PlasticParcels`.
```
Comment (erikvansebille) [inline docs/paper/paper.md:49]: Refer to the documentation website instead of the GitHub repo?

File: docs/paper/paper.md [hunk 4]
```diff
@@ -21,41 +21,40 @@ link-bibliography: false
...
 
-![Particle initialisations maps based on a) current global surface concentrations, b) coastal plastic emissions, c) river plastic emissions, d) fishing activity plastic emissions.\label{fig:initialisation_maps}](initialisation_maps.png){width=80%}
 
-# Usage Examples
-**TODO fill this in once created**
+# Usage Example
 The `PlasticParcels` github repository provides several useful tutorials. Here, we briefly demonstrate how `PlasticParcels` can be used for a microplastic dispersal simulation in the Mediterranean Sea. The tutorial can be found on the `PlasticParcels` github repository, ...
```
Comment (erikvansebille) [inline docs/paper/paper.md:53]: This still needs to be done?

Comment (michaeldenes) [reply docs/paper/paper.md:53]: Yeah, I can create this very quickly if we decide to use the one particle per cell approach that is currently implemented. However, we wanted to do this with n particles per grid cell.

I've now created a script that saves the IDs of the grid cell, so that we can randomly distribute particles across a grid cell. The non-trivial part is how to deal with the curvilinear grid.

I think we can distribute on a [0,1]x[0,1] square, and rotate it like we do with the whole Jacobian approach. I just need to figure out how to do that rotation. Then I can run the example and update the plots.

What do you think?

File: docs/paper/paper.md (hunk 1 above)
Comment (michaeldenes) [reply docs/paper/paper.md:24]: Updated to plasticparcels and parcels

File: docs/paper/paper.md (hunk 2 above)
Comment (michaeldenes) [reply docs/paper/paper.md:41]: Updated to plasticparcels and parcels

File: docs/paper/paper.md (hunk 3 above)
Comment (michaeldenes) [reply docs/paper/paper.md:49]: Updated to documentation page instead of github repo

File: docs/paper/paper.md (hunk 4 above)
Comment (michaeldenes) [reply docs/paper/paper.md]: I've now included a usage example in the docs/paper/paper.md file

**Comment by michaeldenes** [comment]
@erikvansebille , the paper is now in a final draft form (downloadable via the actions tab above).

The only things I can think of are:
a) should we include more references for the line '... users can perform Lagrangian simulations of a wide variety of particulates, such as tuna, plastic, plankton, icebergs, turtles...'?
b) can we remove some of the code in the usage example, and just link to the tutorial and show the plots?

File: docs/paper/schematic.png
Comment (erikvansebille) [inline docs/paper/schematic.png:1]: Why did you change the word 'Kernels' to 'modules' in the revised figure? We use kernels throughout the document, why not here too?

**Review by erikvansebille — COMMENTED** [review]
I added three comments to the paper.md and schematic.png, and also created a PR (#35) with textual and grammar changes/suggestions

File: docs/paper/paper.md [hunk 5]
```diff
@@ -21,41 +21,87 @@ link-bibliography: false
...
+```python
+import plasticparcels as pp
+settings_file = 'docs/examples/example_Italy_coast_settings.json'
+settings = pp.utils.load_settings(settings_file)
+
+settings['simulation'] = {
+    'start_date': datetime.strptime('2019-01-01-00:00:00', '%Y-%m-%d-%H:%M:%S'),
+    'runtime': timedelta(days=30),              # Runtime of simulation
+    'dt_write': timedelta(hours=12),            # Timestep of output
+    'dt_timestep': timedelta(minutes=20),       # Timestep of advection
```
[the remark this answers is not in the document]
Comment (michaeldenes) [reply docs/paper/paper.md:66]: Changed the settings variable names to reflect what we do in parcels

File: docs/paper/paper.md [hunk 6]
```diff
@@ -21,41 +21,87 @@ link-bibliography: false
...
+    }
+```
+
+After turning on/off certain behaviour kernels, we define our plastic particle type, and particle release settings.
```
[the remark this answers is not in the document]
Comment (michaeldenes) [reply docs/paper/paper.md:70]: I've added the four lines used to turn on/off kernels now

File: docs/paper/schematic.png
Comment (michaeldenes) [reply docs/paper/schematic.png:1]: Due to using the figure in a separate presentation. This has been changed back to 'kernels' now.

**Comment by michaeldenes** [comment]
@erikvansebille a last double check review before we release a new version and submit?

File: docs/examples/example_Italy_coast.ipynb
[the remark this answers is not in the document]
Comment (michaeldenes) [reply docs/examples/example_Italy_coast.ipynb:1]: That makes sense! I'll do that now
